### PR TITLE
Paramedics now get nitryl gloves

### PR DESCRIPTION
**Pull request by redmoogle**
### Intent of your Pull Request

Fixes a accidental oversight

Paramedics now get Nitryl gloves for easier fireman carry

#### Changelog

:cl:  
rscadd: Paramedics get Nitryl gloves for fireman carry
/:cl:

### CODE REVIEW

File: code/game/objects/structures/crates_lockers/closets/secure/medical.dm [hunk 1]
```diff
@@ -176,4 +177,4 @@
 	new /obj/item/reagent_containers/dropper(src)
 	new /obj/item/storage/box/syringes/variety(src)
 	new /obj/item/storage/box/beakers/variety(src)
-	new /obj/item/clothing/glasses/science(src)
\ No newline at end of file
+	new /obj/item/clothing/glasses/science(src)
```
Comment (redmoogle) [inline code/game/objects/structures/crates_lockers/closets/secure/medical.dm:180]: Yeah I really dont understand what happened here, it probably just added a extra blank space at the end?

Comment (jcatjmeow) [reply code/game/objects/structures/crates_lockers/closets/secure/medical.dm:180]: your program added a newline at the end of the file automatically.

This is a good thing and you should not worry about it

**Review by jcatjmeow — APPROVED** [review]

**Comment by MegaEmpirical** [comment]
What do nitryl gloves have to do with fireman carry?

**Comment by redmoogle** [comment]
It makes it faster at picking them up, but its not required

**Comment by MegaEmpirical** [comment]
... Why do nitryl gloves make it faster?

**Comment by redmoogle** [comment]
'Expert nanochips' but mainly because it's a little slow so this is to balance it out

**Comment by MegaEmpirical** [comment]
Hmm. Whatever, I'm just happy to be getting gloves.

**Review by JamieD1 — APPROVED** [review]

**Pull request merged**